scripts/aggregate-changelogs/config.yaml

```diff
@@ -14,12 +14,12 @@ repositories:
     category: REST API
   giantswarm/apiextensions:
     category: Management API
-  giantswarm/aqua-app:
-    category: Managed Apps
   giantswarm/aws-admission-controller:
     category: Management API
   giantswarm/azure-admission-controller:
     category: Management API
+  giantswarm/aqua-app:
+    category: Managed Apps
   giantswarm/cert-manager-app:
```

Comment:
So far these were sorted by repository name. But that's not a strict requirement.

Reply:
yes sorry next time i will fix :D